nwbwidgets/ophys/ophys_controllers.py

```diff
@@ -0,0 +1,97 @@
+from typing import Optional
+
+import ipywidgets as widgets
+
+from ..controllers import RotationController, ImShowController, ViewTypeController, MultiController
+
+
+class FrameController(widgets.VBox):
+    controller_fields = ("frame_slider",)
+
+    def __init__(self):
+        super().__init__()
+
+        self.frame_slider = widgets.IntSlider(
+            value=0,  # Actual value will depend on data selection
+            min=0,  # Actual value will depend on data selection
+            max=1,  # Actual value will depend on data selection
+            orientation="horizontal",
+            description="Frame: ",
+            continuous_update=False,
+        )
+
+        self.children = (self.frame_slider,)
+
+
+class PlaneController(widgets.VBox):
```

Comment:
I think defining controllers that contain a single widget is a bit overkill. In my mind, the point of controllers is collections of widgets in a system that is reusable.